### ISSUE TITLE
Crash detected when speed-control is enabled

### ISSUE THREAD

**Issue by Cidana-Developers** [OpenVisualCloud/SVT-AV1#368]
Test Platform: macos
Version:  6a003487ff0bb9eb7e831e276fe2b4ab7bfc79e3
how to build: 
```
./build.sh release
```
how to reproduce:
```
$: ./SvtAv1EncApp -i ../../tempete_cif.y4m  -speed-ctrl 1 -o ./recon.yuv
Encoding          Assertion failed: (sequence_control_set_ptr->cdf_mode == 0 && "use cdf_mode 0"), function signal_derivation_mode_decision_config_kernel_oq, file /Users/mac/work/SVT-AV1/Source/Lib/Common/Codec/EbModeDecisionConfigurationProcess.c, line 1899.
Abort trap: 6
```
Issue is deterministic.

**Comment by JacksonDagger** [comment]
This crash is still present in the master and only affects encoding modes 7 and 8

**Comment by JackZhouVCD** [comment]
@Cidana-Developers can you guys check if the crash is still present? I think it has been addressed by #544 and but I don't have access to a macos machine so I can't confirm.

**Comment by Cidana-Developers** [comment]
Just double checked, no crash raised any more. @JackZhouVCD
